r4/resource-apis/account/Ballerina.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+org = "ballerinax"
+name = "health.fhir.templates.r4.account"
```

Comment:
We decided to prefix health.fhir.templates.international401.<RESOURCE>

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/10d99e6600a555498933a04733525050c16746f1